src/sentry/static/sentry/app/components/events/interfaces/spans/header.tsx

```diff
@@ -551,20 +543,12 @@ const TickLabel = (props: {
 
 const DurationGuideBox = styled('div')<{alignLeft: boolean}>`
   position: absolute;
-
   background-color: ${p => p.theme.white};
   padding: 4px;
-
+  height: 100%;
   border-radius: 3px;
   border: 1px solid rgba(0, 0, 0, 0.1);
-
-  height: 16px;
-
   line-height: 1;
-  vertical-align: middle;
-
-  transform: translateY(50%);
-
   white-space: nowrap;
 
   ${({alignLeft}) => {
```

Comment:
@doralchan may neeed to remove `translateY(50%)` here as well to address the issue when the duration guide box is displayed on the left-hand side
<img width="790" alt="Screen Shot 2019-12-03 at 12 50 58 PM" src="https://user-images.githubusercontent.com/139499/70075929-bfe19d80-15cb-11ea-8e51-63cb51b3b49d.png">

Reply:
![Screen Shot 2019-12-03 at 9 56 17 AM](https://user-images.githubusercontent.com/4830259/70076339-6bcabf00-15b3-11ea-862e-667543809c47.png)